app/Repositories/PhotoRepository.php

```diff
@@ -44,7 +44,11 @@ public function create(array $data, $signupId)
     {
         $postEvent = Event::create($data);
 
-        $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        if (array_key_exists('campaign_id', $data)) {
+            $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        } else {
+            $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        }
```

Comment:
@sbsmith86 oooooo right I got confused and forgot that currently we use `ReportbackRepository.php`, thank you!